nerves_toolchain_ctng/build.sh

```diff
@@ -237,27 +251,7 @@ build_gcc()
         -e 's/^.*\(CT_SAVE_TARBALLS\).*$/\1=y/' \
         $WORK_DIR/build/.config
 
-        $PREFIX $LOCAL_INSTALL_DIR/bin/ct-ng build &
-        local build_pid=$!
-        {
-            while true
-            do
-                sleep 300
-                printf "Not done yet...\r"
-            done
-        } &
-        local runner_pid=$!
-
-        # Wait for the build to finish and get the result
-        wait $build_pid 2>/dev/null
-        local result=$?
-
-        # Stop the runner task
-        kill $runner_pid
-
-        # Wait for the runner task to finish, but ignore the result
-        # in case the runner task ends before the call to wait.
-        wait $runner_pid 2>/dev/null || true
```

Comment:
@fhunleth The keep alive is needed for CircleCI. There is a 10m inactivity timeout. 
```
Too long with no output (exceeded 10m0s)
```

Reply:
Oh. Yeah that is a bummer :(